packages/obonode/obojobo-chunks-materia/viewer-component.js

```diff
@@ -58,7 +79,38 @@ export default class Materia extends React.Component {
 
 			switch (data.type) {
 				case 'materiaScoreRecorded':
-					this.setState({ score: data.score })
+					// this should probably be abstracted in a util function somewhere
+					API.get(
+						`/materia-lti-score-verify?visitId=${this.state.visitId}&nodeId=${this.state.nodeId}`,
+						'json'
+					)
+						.then(API.processJsonResults)
+						.then(result => {
+							const newState = {
+								score: result.score,
+								verifiedScore: true
+							}
+							this.setState({
+								...this.state,
+								...newState
+							})
+
+							const modelId = this.props.questionModel.get('id')
+							const moduleContext = NavUtil.getContext(this.props.moduleData.navState)
+
+							QuestionUtil.setResponse(
```

Comment:
I'd hoped to be able to get to the goal without mangling the `Materia` component or inventing a whole new component, but if the current approach is against the rules I'll probably try to go with option 2 to avoid potentially breaking anything that exists already.